+nosnoc/+objects/Ellipse.m

```diff
@@ -0,0 +1,100 @@
+classdef Ellipse < nosnoc.objects.Object
```

Comment:
maybe this name is too generic. you may want to highlight that it is related to a PDS  and name it PDSObject (not to confuse with PDSObjectS, ofc.)